### DISCUSSION TITLE
Billing issues

### DISCUSSION THREAD

**Discussion by amconteh** [deepgram/community#972] [General help]
Hi, I am not sure who to contact about this.  But I have noticed that my Credits is being used up very rapidly and I am trying to figure out why. 
As far as I can find Deepgram only charges per  minute of audio. For this entire year I have used 24hours 48minutes of audio transcription. Yet i have been charged for $210 of credits and I am not sure why exactly. I would like to figure this out so that If its an issue on my end I can get it fixed.  The only guess I have now is that  it charging per requests.
![Screenshot 2024-10-23 151633](https://github.com/user-attachments/assets/2d2f8acb-ac2c-4d76-b121-948d7b3ed0b1)
 
Anyway help is appreciated.

**Comment by deepgram-community** [comment]
Hey there! It looks like you haven't connected your GitHub account to your Deepgram account. You can do this at https://community.deepgram.com - being verified through this process will allow our team to help you in a much more streamlined fashion.

**Comment by deepgram-community** [comment]
It looks like we're missing some important information to help debug your issue. Would you mind providing us with the following details in a reply?

- The programming language you are working in (e.g. JavaScript, Python).
- The deepgram product you are using (e.g Speech to Text, Agent API)
- A request ID that triggered your error or issue.

**Reply by amconteh** [reply]
Language: Python
Deepgram Product: Speech to Text
Request ID: N/A

**Comment by deepgram-community** [comment]
I'm sorry to hear that you're experiencing issues with your billing. Deepgram charges per minute of audio transcribed, not per request. The cost can vary depending on the model you're using. For example, the Nova model costs $0.006 per minute. 

From the screenshot you've provided, it seems like you've used 24 hours and 48 minutes of audio transcription this year. If you're using the Nova model, the total cost should be around $89.28. 

If you're being charged $210, it's possible that there might be some other factors at play. For example, if you're using other features like data storage or additional services, these could contribute to the cost. 

I would recommend checking your usage details in your Deepgram account to see a breakdown of the charges. 
> Naomi

**Reply by supadupadev** [reply]
1488 minutes times $0.006 is $8.92 not $89.28. If the manual human calculation is 10 times off, no wonder the product coded is 30 times off.

**Comment by jkroll-deepgram** [comment]
Hi @amconteh, I see you also have TTS character usage, and text intelligence input and output token usage. Those will be additional costs, in addition to the audio duration speech-to-text costs. You can see all our pricing info here: https://deepgram.com/pricing

If you provide your Deepgram project ID, we can segment your billing categories in more detail.

**Reply by amconteh** [reply]
@jkroll-deepgram 
Thank you for your suggestion. As for TTS i used for this entire year 5,860 characters when I was using the API program and the chat bot demo. 
Text intelligence input and output i have used: 791 input tokens, 234 output tokens on one day in June 23 2024. I have not used anything since. 


Here is my project id: 5b36f1be-03e5-4578-9e6c-33f5d9289c1f

Thank you for your help
